easybuild/easyblocks/t/tensorflow.py

```diff
@@ -562,6 +562,15 @@ def configure_step(self):
         tensorrt_root = get_software_root('TensorRT')
         nccl_root = get_software_root('NCCL')
 
+        # add path to libnccl.so.2 directory provided by NCCL when both sysroot
```

Comment:
There's an `if` block for NCCL further down, best to keep this stuff together there (and the `libpath` part doesn't get used until a later step anyway)